### PR TITLE
update appversion and myself in maintainers list

### PR DESCRIPTION
**Pull request by zouyee**
/cc @unguiculus

### CODE REVIEW

**Comment by mattfarina** [comment]
cc: @ fiunchinho

File: stable/kube-state-metrics/Chart.yaml [hunk 1]
```diff
@@ -5,11 +5,13 @@ keywords:
 - metric
 - monitoring
 - prometheus
-version: 0.3.1
-appVersion: 1.0.1
+version: 0.3.2
+appVersion: 1.1.0
```
Comment (mattfarina) [inline stable/kube-state-metrics/Chart.yaml]: The container image version in the `values.yaml` file has not increased. So, it is delivering version 1.0.1. To update the app version the container image should also be updated to one with that app version.

**Review by mattfarina — CHANGES_REQUESTED** [review]

File: stable/kube-state-metrics/Chart.yaml (hunk 1 above)
Comment (zouyee) [reply stable/kube-state-metrics/Chart.yaml]: my mistake, thank you.I also update the values of `version` and `appVersion`

**Comment by unguiculus** [comment]
Please clarify why you added yourself to the maintainers list. Are you willing to contribute to the chart in the future and to review PRs?

**Comment by unguiculus** [comment]
cc @fiunchinho

**Comment by fiunchinho** [comment]
LGTM

**Comment by unguiculus** [comment]
Marking stale. Still no response from @zouyee.

**Comment by unguiculus** [comment]
Closing as stale. Feel free to reopen if/when you decide to pick this up again.

**Pull request closed without merging**